src/common/config_opts.h

```diff
@@ -321,7 +321,6 @@ OPTION(mon_cache_target_full_warn_ratio, OPT_FLOAT, .66) // position between poo
 OPTION(mon_osd_full_ratio, OPT_FLOAT, .95) // what % full makes an OSD "full"
 OPTION(mon_osd_backfillfull_ratio, OPT_FLOAT, .90) // what % full makes an OSD backfill full (backfill halted)
 OPTION(mon_osd_nearfull_ratio, OPT_FLOAT, .85) // what % full makes an OSD near full
-OPTION(mon_osd_initial_require_min_compat_client, OPT_STR, "jewel")
```

Comment:
IMO if we want to change this we should switch the config to 'luminous' instead of 'jewel'.  

...but I think we should leave this at jewel since even the latest kernel client doesn't yet support upmap.